spec/features/task_management_spec.rb

```diff
@@ -0,0 +1,50 @@
+require "rails_helper"
+
+RSpec.feature "Task management", :type => :feature do
+
+  scenario "User creates a new valid task" do
+    visit "/tasks"
+
+    fill_in "Todo", :with => "This is a test task"
+    click_button "Save Task"
+
+    expect(page).to have_text("Task successfully created")
+  end
+
+  scenario "User creates a new invalid task" do
+    visit "/tasks"
+
+    click_button "Save Task"
+
+    expect(page).to have_text("Something went wrong!")
+  end
+
+  scenario "User edits a task" do
+    create(:task, id: '1')
+    visit "/tasks/1/edit"
+
+    fill_in "Todo", :with => "This is a test task"
```

Comment:
please don't use the old syntax 
```suggestion
    fill_in 'Todo', with: 'This is a test task'
```